src/main/java/org/openelisglobal/workplan/controller/PrintWorkplanReportController.java

```diff
@@ -107,6 +108,70 @@ public void showPrintWorkplanReport(HttpServletRequest request, HttpServletRespo
         }
     }
 
+    @RequestMapping(value = "/rest/printWorkplanReport", method = RequestMethod.POST)
```

Comment:
i guess you would want to create a separate REST contoller

Reply:
done